cves/CVE-2013-7014.yml

```
@@ -186,8 +194,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: you could write a static analysis tool to look for this type of issue.
```

```suggestion
  answer_note: You could write a static analysis tool to look for this type of issue.
```
This is writing, not just bullet points. Capitalization matters.